app/validators/application_fulfilment_validator.rb

```diff
@@ -22,7 +22,7 @@ def default_url_options
   def perform_validations
     errors = []
 
-    unless Passporting::MeansPassporter.new(record).call
+    unless Passporting::MeansPassporter.new(record).call || evidence_present?
```

Comment:
If this is specific to means_passport, therefore evidence for passporting benefit, should we also be checking that the evidence requested was for benefits? 

Maybe just one for the future once there is an option to unlock the evidence upload step for a different reason, but remain non-means passported and never asked for benefit evidence

Reply:
Yes, not sure currently how we would do this. We can only work with what we have, which is checking if some document has been "stored" (as in successfully being submitted to S3).

We can assume is in the best interest to the provider to upload what they are asked to upload. If they instead upload an IKEA assembly instructions document, application will probably be rejected by the case worker 😅

Reply:
Once we introduce evidence categorisation further validations might be unlocked 👍